presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -373,6 +388,38 @@ Metastore cache only caches schema and table names. Other metadata would be fetc
     hive.metastore-refresh-interval=3d
     hive.metastore-cache-maximum-size=10000000
 
+Extra Hidden Metadata Columns
```

Comment:
I didn't bring it up originally when this topic was added to the documentation, but I am curious about the first word here: why "Extra"? Are there non-extra hidden metadata columns or tables? I'm not requesting a change here at the moment, I'm more curious about why than anything.

Reply:
I kept with the convention of `Extra Hidden Metadata Tables` below this section. I agree the `Extra` phrasing is a little odd. Perhaps `Hidden` would be better or nothing at all.

Reply:
Thanks for the explanation, and the note about hive.rst. I'll worry about it making sense later.